openmdao/docs/features/building_blocks/drivers/doe_driver.rst

```diff
@@ -84,9 +84,12 @@ To illustrate this, we will demonstrate performing a DOE on a model based on the
     :layout: code
 
 In this case, the model itself requires two processors, so in order to run cases
-concurrently we need to allocate at least four processors in total.  With four
-processors we can run two cases at a time, which is done by specifying
-:code:`options['parallel']=2`.
+concurrently we need to allocate at least four processors in total. We can allocate
+as many procs as we have available, however the number of procs must be a multiple
+of the number of procs per model, which is 2 here. Regardless of how many processors
```

Comment:
not sure about mixing in the abbreviated term 'procs' here... would it be better to just consistently use the actual word, processors?

Reply:
Fixed